### ISSUE TITLE
What's the difference between converse-mobile.js and converse.js files

### ISSUE THREAD

**Issue by rheman** [conversejs/converse.js#1125]
Hi everyone. 
Sorry But I just want to know the difference between the following two files.

`converse-mobile.js and converse.js `

best regard.

**Comment by martinnaughton** [comment]
Answered here  https://github.com/conversejs/converse.js/issues/1057

**Comment by rheman** [comment]
Thanks a lot for your answer. But for lastest stable release (3.3.4) converse-mobile.js is not available. So I want to know if for 3.3.4 release the view_mode  is already working ?

Best regard.

**Comment by jcbrand** [comment]
No, the view_mode is not yet working in 3.3.4. You'll need to build your own... by typing `make dist/converse-mobile.js`

**Comment by rheman** [comment]
Thanks a lot . but I have the following error when I try to build. 
```
$ cd converse.js-3.3.4
$ make dist/converse-mobile.js
$ make: *** No rule to make target 'dist/converse-mobile.js'.  Stop.
```

Best regard

**Comment by jcbrand** [comment]
Sorry, my bad. The new view_mode changes were already released in 3.3.0

https://github.com/conversejs/converse.js/releases/tag/v3.3.0

So you don't have to make a separate build.

**Comment by rheman** [comment]
Thanks
